default.json

```diff
@@ -75,10 +75,6 @@
         "/^coverlet\\.collector$/i",
         "/^xunit(\\.|$)/i",

```

Comment:
SHouldn't we remove this rule as a whole, doesn't seem to do anything anymore and the description is "Automatically merge minor and patch updates for Microsoft and trusted, well-known dependencies"

Reply:
Guess we could